.github/workflows/build.yml

```diff
@@ -0,0 +1,61 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - name: macOS 10.15
+            os: macos-10.15
+          - name: Windows 2019 (MSVC)
+            os: windows-2019
+            cc: cl
+            cxx: cl
+
+    runs-on: ${{ matrix.os }}
+    name: ${{ matrix.name }}
+    steps:
+    - name: "Check out repository"
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
I don't think we need this for anything, do we?

Reply:
I don't think so. Removed.